meshio/vtu/_vtu.py

```diff
@@ -75,14 +152,25 @@ def _cells_from_data(connectivity, offsets, types, cell_data_raw):
                     start_cn[items + 1],
                     _vtk_to_meshio_order(types[start], sz, dtype=offsets.dtype) - sz,
                 )
-                cells.append(CellBlock(meshio_type + str(sz), connectivity[indices]))
+                if meshio_type == "polyhedron":
```

Comment:
This could be a bit shortened with
```
data = polyhedron_faces[f"polyhedron{sz}"] if meshio_type == "polyhedron" else connectivity[indices]
cells.append(CellBlock(meshio_type + str(sz), data)
```